hs-init.hs

```diff
@@ -508,7 +514,11 @@ customizeLicense l t nm year
 -- This is needed to be able to call commands by writing strings.
 instance (a ~ Text, b ~ ()) => IsString ([a] -> IO b) where
   fromString "cd" [arg] = setCurrentDirectory $ T.unpack arg
-  fromString cmd args   = callCommand $ showCommandForUser cmd (map T.unpack args)
+  fromString cmd args   = callProcess cmd (map T.unpack args)
```

Comment:
Could you please clarify the reason for changing `callCommand showCommandForUser` with `callProcess`?

Reply:
Despite the docs claiming otherwise, `showCommandForUser` doesn't actually produce a string that works on windows.  The problem seems similar to https://github.com/haskell/process/issues/51